ODB Classes/ODBTableViewController.m

```diff
@@ -183,11 +183,23 @@ - (BOOL)listDirectoryAtPath:(NSString *)path {
             if (files) {
                 self.currentFiles = nil;
                 self.currentFiles = [NSMutableArray arrayWithCapacity:files.count];
-                for (NSString *file in files) {
-                    if (self.allowedFileTypes.count == 0 || [self.allowedFileTypes containsObject:[file pathExtension]]) {
+                for (NSDictionary *file in files) {
```

Comment:
It shouldn't be a dictionary.
It wont work. 
each file of the files is DBFILESMetadata.